operator_csv_libs/tests/csv_test.py

```diff
@@ -442,6 +447,19 @@ def test___update_operand_images(self):
         testcsvWithoutParams._update_operand_images()
         self.assertEqual(testcsvWithoutParams.csv, TEST_DUMMY_CSV)
 
+    def test__get_deployments(self):
+        # Read in a sample CSV
+        with open(THIS_DIR + '/test_files/valid_csv.yaml', 'r') as stream:
+            sample = yaml.safe_load(stream)
+
+        csv = ClusterServiceVersion(sample)
+        deployments = csv.get_operator_deployments()
+        ## Assert that dpeloyment is as expected
```

Comment:
value we get is never asserted to make sure it is what we expect.

Reply:
Thanks for reviewing and noticing @eveliososa 
Addressed